syntax = "proto3";

option java_package = "com.huzhengxing.grpcinterface.api";
option java_outer_classname = "HelloWorldService";

package helloworld;

service HelloWorld {
  rpc SayHello (HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
  string name = 1;
}

message HelloResponse {
  string message = 1;
}